grama/eval_pnd.py

```diff
@@ -105,8 +103,9 @@ def eval_pnd(model, df_train, df_test, sign, n=int(1e4), seed=None):
     sds = []
     vars = df_train.columns.values
     input = model.var
-    outputs = [value for value in vars if value not in input]
+    outputs = [key for key in signs.keys() if key in df_test.columns.values]
     length = int(len(model.out) / 2)
+    signs = [value for value in signs.values()]
 
     for i, value in enumerate(model.out):
```

Comment:
This targets all the outputs in the model, not the outputs specified by the argument `signs`. Should modify to target only the specified outputs.